nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/status/history/EmbeddedQuestDbRolloverHandlerTest.java

```diff
@@ -149,43 +247,39 @@ private void givenTableIsCreated(final QuestDbContext dbContext) throws Exceptio
         dbContext.getCompiler().compile(CREATE_TABLE, dbContext.getSqlExecutionContext());
     }
 
-    private void givenTableIsPopulated(final List<Long> givenMeasurementTimes) {
+    private void givenTableIsPopulated(final String... dates) throws Exception {
+        int value = 0;
+
+        for (final String date : dates) {
+            givenTableIsPopulated(date, value++);
+        }
+    }
+
+    private void givenTableIsPopulated(final String date, final int value) throws Exception {
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss z").withZone(ZoneId.systemDefault());
```

Comment:
As the format has zone part (`z`), I don't think `withZone()` could have any effect.
Also in line 274.